common/persistence/dataManagerInterfaces.go

```diff
@@ -1845,6 +1856,8 @@ type (
 		Payload   []byte    `json:"message_payload"`
 	}
 
+	QueueMessageList []*QueueMessage
```

Comment:
Is this change metrics related? I would consider pulling it out separately?

Reply:
It is metric related. I need a method defined on the response, and that could only be done on the type.

Reply:
ah, yeah, sorry, need more coffee, totally makes sense